frontend/sozisel-app/src/components/SessionsList/SessionsList.tsx

```diff
@@ -0,0 +1,186 @@
+import "./SessionsList.scss";
+
+import { BaseSyntheticEvent, ReactElement, useMemo, useState } from "react";
+import {
+  CircularProgress,
+  InputAdornment,
+  List,
+  TextField,
+} from "@material-ui/core";
+import MuiAlert, { AlertProps } from "@material-ui/lab/Alert";
+import SessionFilters, {
+  SessionFiltersSchema,
+} from "./SessionFilters/SessionFilters";
+import {
+  SessionStatus,
+  useDeleteSessionMutation,
+  useSearchSessionsQuery,
+} from "../../graphql";
+
+import ClearIcon from "@material-ui/icons/Clear";
+import MainNavbar from "../MainNavbar/MainNavbar";
+import SearchIcon from "@material-ui/icons/Search";
+import SessionCard from "./SessionCard/SessionCard";
+import Snackbar from "@material-ui/core/Snackbar";
+import { debounce } from "debounce";
+import { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+
+function Alert(props: AlertProps) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+export default function SessionsList(): ReactElement {
+  const { t } = useTranslation("common");
+  const [filters, setFilters] = useState<SessionFiltersSchema>({
+    status: SessionStatus.Any,
+    templateId: undefined,
```

Comment:
if you don't specify the `templateId: undefined` it should automatically get undefined value.